templates/index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+
+<!-- generated by dartdoc -->
+
+<html>
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Package {{packageName}}</title>
+    <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.1/css/bootstrap.min.css" rel="stylesheet" media="screen">
```

Comment:
dbc: it would be nice to have the base for the boostrap cdn be a template var. I.e., 

    https://maxcdn.bootstrapcdn.com/bootstrap/3.3.1